docs/gettingstarted/mk8s/k8s-microk8s-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Clustering SNMP with microk8s
+
+## Master node
+
+1. Setup master node using this [guide](./k8s-microk8s.md). 
+
+2. Generate joining token for replicas:
+
+```bash
+microk8s add-node
+```
+
+3. After running `add-node` on stdout you will got a commamd (like `microk8s join <master_node>:25000/<token>`) that you need to remember.
+
+## Replicas nodes
+
+1. Installing microk8s on replica node: 
+
+```bash
+sudo snap install microk8s --classic --channel=1.30/stable
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+su - $USER
+```
+
+2. After run `add-node` on **master node** you will got on stdout command that you need to run on your **replica node**:
+
+```bash
+microk8s join <master_node>:25000/<token>
+```
+
+3. Check that replicas joined cluster:
+
+```bash
+microk8s kubectl get nodes
```

Comment:
maybe you can add below the example how the output should look if everything is set fine